src/main/scala/uk/gov/nationalarchives/omega/api/ApiServiceApp.scala

```diff
@@ -34,6 +34,8 @@ object ApiServiceApp extends IOApp {
   implicit val loggerFactory: LoggerFactory[IO] = Slf4jFactory[IO]
   implicit val logger: SelfAwareStructuredLogger[IO] = LoggerFactory[IO].getLogger
 
+  val applicationId = "PACS001"
+
```

Comment:
Does Omega maintain a list of all recognised `applicationId` so the system would respond to only systems that are registered?  If so, should this be configurable and this would be defined in service configuration perhaps?